src/App.js

```diff
@@ -1,11 +1,24 @@
 import React from "react";
+import { Route } from 'react-router-dom';
 import Header from "./components/Header.js";
 
 
+
+import WelcomePage from './components/WelcomePage';
+import CharacterList from './components/CharacterList';
+import SearchForm from './components/SearchForm';
+
 export default function App() {
   return (
     <main>
-      <Header />
-    </main>
-  );
+    <Header />
+        <div>
+            <Route exact path="/" component={WelcomePage} />
+            <Route path="/characters" component={CharacterList} />
+            <Route path="/search" component={SearchForm} />
```

Comment:
The `<SearechForm/>` component already exists inside of `<CharacterList/>`. It also takes in props to handle the input form.

Because the `<SearchForm/>` needs those props, I would remove this `<Route/>`. If someone manually links themself here and types into the input, the webpage throws an error 😮